portfolio/src/main/webapp/index.html

```diff
@@ -2,74 +2,133 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Kanupriya's Portfolio</title>
     <link rel = "stylesheet" href = "style.css">
     <script src = "script.js"></script>
   </head>
   <body>
-
-        <div class = "container navbar">
+      
```

Comment:
trailing white spaces? If so I think it'll be nice to remove them :)

Reply:
This may actually be a tab.

Reply:
Fixed this